ansible/docker/rally/Makefile

```diff
@@ -1,4 +1,7 @@
 image_name=ovn-scale-test-rally
 
 image:
+	tar cvzf /tmp/ovn-scale-test.tgz ../../../../ovn-scale-test
```

Comment:
I'm not quite following what these changes are doing, can you explain? What I think is happening is this:

1. You are tar'ing up the existing ovn-scale-test directory, then untar'ing it in /tmp
2. the "docker build" command is using this directory from /tmp to build
3. You then rm the directory from /tmp

Is that right?